ios/Membrane.swift

```diff
@@ -618,6 +618,79 @@ class Membrane: RCTEventEmitter, MembraneRTCDelegate {
     }
     resolve(nil)
   }
+    
+  private func getMapFromStatsObject(obj: RTCInboundStats) -> [String: Any] {
+      var res: [String:Any] = [:]
+      res["kind"] = obj.kind
+      res["jitter"] = obj.jitter
+      res["packetsLost"] = obj.packetsLost
+      res["packetsReceived"] = obj.packetsReceived
+      res["bytesReceived"] = obj.bytesReceived
+      res["framesReceived"] = obj.framesReceived
+      res["frameWidth"] = obj.frameWidth
+      res["frameHeight"] = obj.frameHeight
+      res["framesPerSecond"] = obj.framesPerSecond
+      res["framesDropped"] = obj.framesDropped
+      
+      return res
+  }
+
+  private func getMapFromStatsObject(obj: RTCOutboundStats) -> [String: Any] {
+      var innerMap: [String: Double] = [:]
+      if let bandwidth = obj.qualityLimitationDurations?.bandwidth {
```

Comment:
https://www.hackingwithswift.com/example-code/language/what-is-the-nil-coalescing-operator

Reply:
Done.